net-libs/libnice/libnice-0.1.13.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A library for establishing peer-to-peer UDP data streams"
+DESCRIPTION="Libnice is an implementation of the IETF's Interactive .
+Connectivity Establishment (ICE) standard (RFC 5245) and the Session Traversal .
+Utilities for NAT (STUN) standard (RFC 5389) .
+
+ICE is useful for applications that want to establish peer-to-peer UDP .
+data streams. It automates the process of traversing NATs and provides .
+security against some attackes. It also allows applications to create .
+reliable streams using a TCP over UDP layer."
+HOMEPAGE="http://nice.freedesktop.org/wiki"
+COPYRIGHT="2006-2011 Collabora Ltd.
+    2006-2011 Nokia Corporation"
+LICENSE="MPL v1.1
+    GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://nice.freedesktop.org/releases/libnice-0.1.13.tar.gz"
+CHECKSUM_SHA256="61112d9f3be933a827c8365f20551563953af6718057928f51f487bfe88419e1"
+PATCHES="libnice_x86-0.1.13.patchset"
+
+ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+    libnice$secondaryArchSuffix = $portVersion
+    devel:libnice$secondaryArchSuffix = $portVersion
```

Comment:
I tried to move it to devel, but then haikuporter will complain that it cannot find a provide for devel:libnice_x86 although it is included under devel already, so I left the original one intact at the moment until I figure out why.